### ISSUE TITLE
RTL Content styling adjustments

### ISSUE THREAD

**Issue by mlewand** [ckeditor/ckeditor5#1976]
## Is this a bug report or feature request? (choose one)

🐞 Bug report

## 💻 Version of CKEditor

CKEditor v5 @ 12.4.0

## 📋 Steps to reproduce

This is an umbrella ticket.

There are several content styling that should be still tweaked for improved RTL support.

* Side images should be aligned to the left side.
* List styles should be adjusted, so there's some margin on a right hand side.

![CKEditor in RTL mode with several content styling issues](https://user-images.githubusercontent.com/5353898/63329136-b0e32200-c331-11e9-9e93-0984a16631e6.png)

Also #1151 lists a few content-related things to be ironed out.

You can use https://ckeditor.com/docs/ckeditor5/latest/features/ui-language.html to test the current version (will be published in couple days at a time of writing).

**Comment by mlewand** [comment]
I checked our manual tests and "List styles should be adjusted, so there's some margin on a right hand side." is caused in docs by Umberto, so that's an upstream not related to the editor implementation.

**Comment by mlewand** [comment]
Closing this issue we don't need an umbrella ticket for tracking this, as there are not that many content issues after all (lists for example was in our docs styling, not the editor). I'll open a dedicated issue for side image styling.
